src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.component.js

```diff
@@ -30,16 +32,50 @@ const EnrollmentPagePlain = ({ classes, enrollmentPageStatus }) => (<>
         }
 
         {
-            enrollmentPageStatus === enrollmentPageStatuses.LOADING &&
-                <div className={classes.loadingMask}>
-                    <LoadingMaskForPage />
+            enrollmentPageStatus === enrollmentPageStatuses.MISSING_PROGRAM_SELECTION &&
+            <IncompleteSelectionsMessage>
+                {i18n.t('Choose program to view more information')}
+            </IncompleteSelectionsMessage>
+        }
+
+        {
+            enrollmentPageStatus === enrollmentPageStatuses.MISSING_PROGRAM_CATEGORIES_SELECTION &&
+            <IncompleteSelectionsMessage>
+                {i18n.t('MISSING CATEGORIES NEEDS BETTER TEXT')}
+            </IncompleteSelectionsMessage>
+        }
+
+        {
+            enrollmentPageStatus === enrollmentPageStatuses.MISSING_ENROLLMENT_SELECTION &&
+            <IncompleteSelectionsMessage>
+                {i18n.t('Choose enrollment to view more information')}
+            </IncompleteSelectionsMessage>
+
+        }
+
+        {
+            enrollmentPageStatus === enrollmentPageStatuses.TRACKER_PROGRAM_WITH_ZERO_ENROLLMENTS_SELECTED &&
+            <IncompleteSelectionsMessage>
+                {i18n.t('There are no enrollments for this program.')}
```

Comment:
Suggestion from karoline. => "There are no enrollments for {teiName} in the selected program"

Reply:
The suggestion sounds good to me